src/earn/utils/getRoundedAmount.ts

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
any reason you need this over `internal/utils/getRoundedAmount`? can we dedupe between the two?